src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -115,6 +115,75 @@ public String toString() {
         return s.toString();
     }
 
+    /**
+     * Copied from scm API plugin.
```

Comment:
This is not maintainable; better to depend on https://github.com/jenkinsci/scm-api-plugin/pull/282 which migrates this method (compatibly!) to EE 9.